From 2c523957d8f96abf7910f5efb04bb4ee243fcd03 Mon Sep 17 00:00:00 2001
From: Hou Zhiqiang <Zhiqiang.Hou@nxp.com>
Date: Wed, 18 Sep 2019 15:44:33 +0800
Subject: [PATCH 119/245] Revert "PCI: Mark NXP LS1088 to avoid bus reset bus"

commit 18f70a1ffe2338c46225dd53769c3d30b63fa7fa from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

This reverts commit 0532bd0186fc02c32acc5eef04a8efc24cab2f8a.

The link down issue has been fixed, so revert this workaround.

Signed-off-by: Hou Zhiqiang <Zhiqiang.Hou@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/pci/quirks.c | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/drivers/pci/quirks.c b/drivers/pci/quirks.c
index 3eadad1062a7..70366add1794 100644
--- a/drivers/pci/quirks.c
+++ b/drivers/pci/quirks.c
@@ -3389,13 +3389,6 @@ DECLARE_PCI_FIXUP_HEADER(PCI_VENDOR_ID_ATHEROS, 0x003c, quirk_no_bus_reset);
 DECLARE_PCI_FIXUP_HEADER(PCI_VENDOR_ID_ATHEROS, 0x0033, quirk_no_bus_reset);
 DECLARE_PCI_FIXUP_HEADER(PCI_VENDOR_ID_ATHEROS, 0x0034, quirk_no_bus_reset);
 
-/*
- * NXP (Freescale Vendor ID) LS1088 chips do not behave correctly after
- * bus reset. Link state of device does not comes UP and so config space
- * never accessible again.
- */
-DECLARE_PCI_FIXUP_HEADER(PCI_VENDOR_ID_FREESCALE, 0x80c0, quirk_no_bus_reset);
-
 /*
  * Root port on some Cavium CN8xxx chips do not successfully complete a bus
  * reset when used with certain child devices.  After the reset, config
@@ -3403,13 +3396,6 @@ DECLARE_PCI_FIXUP_HEADER(PCI_VENDOR_ID_FREESCALE, 0x80c0, quirk_no_bus_reset);
  */
 DECLARE_PCI_FIXUP_HEADER(PCI_VENDOR_ID_CAVIUM, 0xa100, quirk_no_bus_reset);
 
-/*
- * NXP (Freescale Vendor ID) LS1088 chips do not behave correctly after
- * bus reset. Link state of device does not comes UP and so config space
- * never accessible again.
- */
-DECLARE_PCI_FIXUP_HEADER(PCI_VENDOR_ID_FREESCALE, 0x80c0, quirk_no_bus_reset);
-
 static void quirk_no_pm_reset(struct pci_dev *dev)
 {
 	/*
-- 
2.17.1

